src/Equinor.ProCoSys.IPO.WebApi/Controllers/Scope/ScopeController.cs

```diff
@@ -26,17 +26,21 @@ public class ScopeController : ControllerBase
         /// <param name="plant"></param>
         /// <param name="projectName"></param>
         /// <param name="startsWithCommPkgNo"></param>
+        /// <param name="itemsPerPage"></param>
+        /// <param name="currentPage"></param>
         /// <returns>All ProCoSys commpkgs that match the search parameters</returns>
         [Authorize(Roles = Permissions.COMMPKG_READ)]
         [HttpGet("CommPkgs")]
-        public async Task<ActionResult<List<ProCoSysCommPkgDto>>> GetCommPkgsInProject(
+        public async Task<ActionResult<ProCoSysCommPkgSearchDto>> GetCommPkgsInProject(
```

Comment:
NBNB! This is a breaking change for client!! How will you solve this for existing client?

Reply:
Will merge this PR at the same time as Sverre takes out UI was my first thought. Now I see that that will result in QP not working if UI is put there a long time before being taken to prod.

What is the procedure for cases like this?

Reply:
It depends .... we must discuss